src/main/java/com/project/trackfit/control/MainController.java

```diff
@@ -1,26 +1,36 @@
 package com.project.trackfit.control;
 
-import com.project.trackfit.model.Customer;
-import com.project.trackfit.service.CustomerService;
+import com.project.trackfit.model.User;
+import com.project.trackfit.service.UserService;
 import lombok.AllArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 @RestController
 @AllArgsConstructor
-@RequestMapping("api/customers")
+@RequestMapping("api/v1")
 public class MainController {
```

Comment:
```suggestion
public class CustomerController {
```

Reply:
Made the Change
1. @RequestMapping("api/customers") -> @RequestMapping("api/v1")
2. MainController -> UserController (I think it's best to mention users instead of customers)